tests/tests/test_mender_connect.py

```diff
@@ -268,6 +270,18 @@ def assert_env(self, docker_env):
         assert (
             docker_env.devconnect is not None
         ), "docker_env must have a set up 'devconnect' instance"
+        proxy_connected_timeout_s = 15 * 60
+        docker_env.device.run(
+            """dbus-send --print-reply --system \\
+              --dest=io.mender.AuthenticationManager \\
+              /io/mender/AuthenticationManager \\
+              io.mender.Authentication1.FetchJwtToken""",
+            wait=proxy_connected_timeout_s,
+        )
+        output = docker_env.device.run(
+            "dbus-send --system --dest=io.mender.AuthenticationManager --print-reply /io/mender/AuthenticationManager io.mender.Authentication1.GetJwtToken"
+        )
```

Comment:
Why do you need the JWT?

Reply:
this is something that (according to my tests) helps in stabilizing the test. I do not need it, but we have to be sure that it is all connected and ready to go before we start this test.

Reply:
and if you ask about printing it: I want to see for the logging purposes what DBus returned